### PR TITLE
Update torch.inverse to torch.linalg.inv

### PR DESCRIPTION
**Pull request by wangshuai09**
#### Type of change

- [x] 🔬 New feature (non-breaking change which adds functionality)

#### Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Did you update CHANGELOG in case of a major change?

#### Why
[torch.inverse()](https://pytorch.org/docs/1.8.0/generated/torch.inverse.html?highlight=inverse#torch.inverse) is deprecated since torch 1.9.0, change that to [torch.linalg.inv()](https://pytorch.org/docs/1.8.0/linalg.html#torch.linalg.inv) following the offical.

### CODE REVIEW

**Review by edgarriba — APPROVED** [review]
LGTM. Before merging, can you extend at least one of the existing tests that shows clearly potential bugs

File: tests/utils/test_helpers.py [hunk 1]
```diff
@@ -82,6 +82,14 @@ def test_jit(self, device, dtype):
         op = _torch_inverse_cast
         op_jit = torch.jit.script(op)
         assert_close(op(x), op_jit(x))
+    
+    def test_not_invertible(self, device, dtype):
+        x = torch.tensor([[0.0, 0.0], [0.0, 0.0]], device=device, dtype=dtype)
+    
+        try:
```
Comment (edgarriba) [inline tests/utils/test_helpers.py:89]: the pytest way to test exceptions is via `pytest.raise` context manager
- https://docs.pytest.org/en/7.1.x/how-to/assert.html#assertions-about-expected-exceptions
- https://github.com/kornia/kornia/blob/main/tests/color/test_gray.py#L22

Comment (wangshuai09) [reply tests/utils/test_helpers.py:89]: Thank you very much. I will learn about that.

Comment (edgarriba) [reply tests/utils/test_helpers.py:89]: besides, i'd like to see a unit test (with corner case scenario) proving that any of our api is affected when the situation you exposed in the original issue happens

Comment (wangshuai09) [reply tests/utils/test_helpers.py:89]: My apologize, I used the template to generate the description, but an irrelevant issue was automatically associated. Do i still need to write a unit test？

Comment (edgarriba) [reply tests/utils/test_helpers.py]: ok, apologies -- i thought this PR was attempting to fix https://github.com/kornia/kornia/issues/2822 for the cross operator

**Review by edgarriba — APPROVED** [review]

**Pull request merged**